app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+# Top-level documentation comment for Like module
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] @Uthmanbello great work on your association however your belongs_to association is wrong, you should reference using the author and then state the class like you did in the comment model `  belongs_to :author, class_name: 'User'`